vendored/sqlite3-parser/src/parser/ast/fmt.rs

```diff
@@ -792,6 +792,7 @@ impl ToTokens for Operator {
             Self::ArrowRightShift => s.append(TK_PTR, Some("->>")),
             Self::BitwiseAnd => s.append(TK_BITAND, None),
             Self::BitwiseOr => s.append(TK_BITOR, None),
+            Self::BitwiseNot => s.append(TK_BITNOT, None),
```

Comment:
@penberg do you think we should start adding some kind of `// <LIMBO_MODIFICATION>` comment tags to things we change in the vendored version? I didn't do this before but started thinking it might be useful

Reply:
No, I think it's OK like this. We could, of course, try to also submit this stuff to upstream.